tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -53,19 +55,22 @@ public partial class PackageViewModel : ObservableObject
     /// Indicates if a package is selected
     /// </summary>
     [ObservableProperty]
+    [NotifyPropertyChangedFor(nameof(ButtonAutomationName))]
     private bool _isSelected;
 
     public PackageViewModel(
         ISetupFlowStringResource stringResource,
         IWindowsPackageManager wpm,
         IWinGetPackage package,
         IThemeSelectorService themeSelector,
+        IScreenReaderService screenReaderService,
```

Comment:
I'm realizing we're adding more and more parameters to our constructors, for this one I assume, some of these will be spit up as part of the refactor efforts once we bring in more package managers?

Reply:
Good observation and you are right, too many injected arguments is a good sign that it's time to refactor this service 🔔 I will consider this as I start refactoring the code, it's unclear to me yet which of the services I can aggregate here into a new one.

Reference: [Refactoring to Aggregate Services](https://blog.ploeh.dk/2010/02/02/RefactoringtoAggregateServices/)